lib/models/builder.js

```diff
@@ -209,6 +210,12 @@ class Builder extends CoreObject {
       }
     }
 
+    this.ui.startProgress(progress.format(progress()));
+
+    const interval = setInterval(() => {
```

Comment:
mind renaming it to `intervalId` to be more explicit?

Reply:
Good call, making it clear that we are differentiating between the actual interval, and the "interval cookie" provided by `setInterval` makes sense.